src/ImportDefinitionsBundle/Provider/ImportDataSet.php

```diff
@@ -0,0 +1,109 @@
+<?php
+/**
+ * Import Definitions.
+ *
+ * LICENSE
+ *
+ * This source file is subject to the GNU General Public License version 3 (GPLv3)
+ * For the full copyright and license information, please view the LICENSE.md and gpl-3.0.txt
+ * files that are distributed with this source code.
+ *
+ * @copyright  Copyright (c) 2016-2018 w-vision AG (https://www.w-vision.ch)
+ * @license    https://github.com/w-vision/ImportDefinitions/blob/master/gpl-3.0.txt GNU General Public License version 3 (GPLv3)
+ */
+
+namespace ImportDefinitionsBundle\Provider;
+
+class ImportDataSet implements ImportDataSetInterface
```

Comment:
do we actually need this class? It does the same as the actual Iterator you would have to pass.

Reply:
It implements count(). Passing iterator would result in "method count() not implemented" error.

Reply:
that is true :)